### ISSUE TITLE
Android AGP 8.0 support

### ISSUE THREAD

**Issue by orkun1675** [kekland/croppy#30]
Thanks for the plugin!

The plugin should define a namespace in its gradle file.

I'm currently getting:

```
Failed to query the value of property 'buildFlowServiceProperty'.
> Could not isolate value org.jetbrains.kotlin.gradle.plugin.statistics.BuildFlowService$Parameters_Decorated@24ba30ee of type BuildFlowService.Parameters
   > A problem occurred configuring project ':croppy'.
      > Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
         > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

           If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.
```

**Comment by orkun1675** [comment]
https://github.com/kekland/croppy/pull/31 resolves this for me.

**Comment by kekland** [comment]
Hi! Thanks for opening a pull request. I'm currently extremely busy with other work but I'll check on it next week

**Comment by kekland** [comment]
Hi! Thanks a lot. Merged #31 now (although removed the part with color changes; these will be refactored differently). Closing this, feel free to reopen if issues persist.
